src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -120,12 +123,20 @@ private boolean checkEnabled(
             enabled =
                     featureToggle.getStrategies().stream()
                             .anyMatch(
-                                    as ->
-                                            getStrategy(as.getName())
-                                                    .isEnabled(
-                                                            as.getParameters(),
-                                                            enhancedContext,
-                                                            as.getConstraints()));
+                                    strategy -> {
+                                        Strategy configuredStrategy = getStrategy(strategy.getName());
+                                        if (configuredStrategy == UNKNOWN_STRATEGY) {
+                                            LOGGER.warn(
```

Comment:
I think this is fine, it is good to inform the user/developer that there is something missing, but would just like to get someones second opinion: 

Is it ok to log this error every single time "isEnabled" is called? Or should we try to only log it once? I am quite not sure what the right answer is, but I can easily see the application log being blown up by a simple miss-configuration in the UI.

Reply:
I think logging every time is preferred to display why something isn't working. Especially to get in this state, it isn't a simple typo in the Admin UI (you have to create a new strategy, then assign it as an activation to a toggle)

If a team does this commonly (I can't imagine why) They could mute the logger completely.

Reply:
thanks. this is a valid point.